src/Tgstation.Server.Host/IO/Console.cs

```diff
@@ -7,7 +7,10 @@
 
 namespace Tgstation.Server.Host.IO
 {
-	/// <inheritdoc />
+	/// <summary>
+	/// Implementation of <see cref="Tgstation.Server.Host.IO.IConsole"/>,
```

Comment:
Missed this one

Reply:
I'm so close to pulling the trigger on Rider I swear to god